ynput/OpenPype
# Global: color v3 global oiio transcoder plugin

## Brief description
Implements possibility to use `oiiotool` to transcode image sequences from one color space to another(s).

## Description
Uses collected `colorspaceData` information about source color spaces, these information needs to be collected previously in each DCC interested in color management.
Uses profiles configured in Settings to create single or multiple new representations (and file extensions) with different color spaces.

New representations might replace existing one, each new representation might contain different tags and custom tags to control its integration step.

## Additional info
This PR is built on top of https://github.com/ynput/OpenPype/pull/4270.

## Testing notes:
1. Set File rules in DCC which has currently Color management implemented (Nuke??) `project_settings/nuke/imageio/file_rules/rules`
![2023-01-06 18_33_01-Window](https://user-images.githubusercontent.com/4457962/211068008-8fe509e3-f93b-456f-9a26-e97dc3be1554.png)
3. Configure output definitions for converted files `project_settings/global/publish/ExtractColorTranscode`

![2023-01-06 18_39_40-OpenPype Settings](https://user-images.githubusercontent.com/4457962/211068068-7257d3c2-ce29-4b9d-921d-5329e4ad2b74.png) (this says "For all representations from Nuke (which have `colorspaceData` prefilled) create new representation with files with same extension and convert it to `ACES` colorspace and send it to `ExtractReview`)

5.  Check if final files are in proper color space.
